lib/src/cupertino/cupertino_controls.dart

```diff
@@ -754,6 +756,9 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position - const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.max(skip, beginning)));
+    Future.delayed(const Duration(milliseconds: 1000), () {
```

Comment:
Add comments above this line explaining why there needs to be a one second delay before setting the playback speed.

That way, any future dev beyond yourself will be better able to understand why its being done this way.

Reply:
Hi Diego , 
I've added the added explaining the reason for adding a delay of 1 second in the code.
Kindly review it.
Thanks and Regards